### DISCUSSION TITLE
posts/comparing-llama3-phi3-gemma/

### DISCUSSION THREAD

**Discussion by giscus** [nithinbekal/nithinbekal.github.io#35] [Comments]
# posts/comparing-llama3-phi3-gemma/

Nithin Bekal's blog about programming - Ruby, Rails, Vim, Elixir.

https://nithinbekal.com/posts/comparing-llama3-phi3-gemma/

**Comment by dibu28** [comment]
have you tyired to run phi3:14b ? on 1080ti? 11GB vram should be enough

**Reply by nithinbekal** [reply]
@dibu28 I finally tried it out today, and it runs really well. I saw around 31 tokens/s on the 1080Ti.
